### ISSUE TITLE
Lab results included in Condition Summary section fail accessibility tests

### ISSUE THREAD

**Issue by austin-hall-skylight** [CDCgov/phdi#2421]
## Description

Lighthouse tests are failing because lab result headers are 2 sizes larger than the condition summary header.

<img width="1308" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNHQrQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--55ae02839c043dcd1fbd26cb133e11111ba56b10/image.png" alt="image.png" />

## Steps to reproduce
View [this ecr summary](http://localhost:3000/view-data?id=1.2.840.114350.1.13.297.3.7.8.688883.567479#ecr-summary) and run lighthouse accessibility tests.

## Expected Result

Tests pass with no audit errors.

## Actual result

I get the error 
`Heading elements are not in a sequentially-descending order`

#### Priority
- [x] **Low priority** _The bug does not have to be fixed immediately. High and medium priority bugs should be addressed first._
- [ ] **Medium priority** _Typically, medium-priority bugs do not affect customers directly and therefore can be fixed in the normal course of testing and development._
- [ ] **High priority** _High priority bugs must be addressed immediately. They often affect the app’s functionality and impact customers and their user experience. They must take priority._

## Notes
